### DISCUSSION TITLE
instant crashes?

### DISCUSSION THREAD

**Discussion by Eggayne** [maltejur/discord-screenaudio#203] [Q&A] [answered]
whenever i open, it opens normally however it doesnt show anyone other users as online.

then shortly after opening, 
![image](https://github.com/maltejur/discord-screenaudio/assets/149186857/4bc69fc0-ba19-44b5-b9f0-b47f01af3a8a)
this happens

ive tried reloading and also reinstalling.

i am on the most recent version

does anyone have any idea what this issue is, or what i could do to figure out the root of the problem

**Comment by Eggayne** [comment]
solved: reinstall again

**Comment by maltejur** [comment, marked as answer]
This was fixed in [v1.9.2](https://github.com/maltejur/discord-screenaudio/releases/tag/v1.9.2)
